fastapi/encoders.py

```diff
@@ -44,7 +44,7 @@ def jsonable_encoder(
     if isinstance(obj, BaseModel):
         encoder = getattr(obj.__config__, "json_encoders", {})
         if custom_encoder:
-            encoder.update(custom_encoder)
+            encoder.copy().update(custom_encoder)
```

Comment:
If I'm reading the code correctly, `encoder.copy()` would create another object. Then it would update that new object, but it would not be used anywhere else. Right?